cadcutils/cadcutils/net/auth.py

```diff
@@ -100,33 +178,37 @@ def get_cert(cert_server=None,
     cert_server = cert_server is None and CERT_SERVER or cert_server
     cert_endpoint = cert_endpoint is None and CERT_ENDPOINT or cert_endpoint
 
-    username, passwd = get_user_password(cert_server)
+    username = user
+    if username is None:
+        sys.stdout.write("CADC user ID: ")
+        username = sys.stdin.readline().strip('\n')
+
+    username, passwd = get_user_password(cert_server, username=username)
 
     url = "http://{0}/{1}".format(cert_server, cert_endpoint)
     resp = requests.get(url, params=kwargs, auth=(username, passwd))
     resp.raise_for_status()
```

Comment:
why not a 'check_status' like is done in get_data?   and why is there a check_status method in get_data btw?